charts/blockchain-explorer/templates/explorer-config.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: explorer-config
+  namespace: explorer
+  annotations:
+    "description": "Configuration of Blockchain Explorer"
+data:
+  config.json: |-
+{{ required "quorumconfig must be set" .Values.quorumConfig | indent 4 }}
+
+  .env.production: |-
```

Comment:
why is it named production?

Reply:
The quorum explorer today just differentiates in ".env.local" if run with "npm run dev" and ".env.production" if run with "npm run build & npm run start".